install/03_rbac.yaml

```diff
@@ -110,6 +110,14 @@ rules:
     - update
     - patch
     - delete
+- apiGroups:
+    - config.openshift.io
+  resources:
+    - infrastructures
```

Comment:
i looked through the code but i couldn't see where this operator is attempting to use an infrastructure object, do we need this permission?

Reply:
Ah sorry, that was in the previous PR, it's being called here https://github.com/openshift/cluster-autoscaler-operator/pull/235/files#diff-c2a06a8d073270721be5c29036cf48fc7b1cffe05223c2f277fbe46bc967180aR46

Reply:
ahh, thank you very much. i missed that call.